identifiers/country-et/constituencies.csv

```diff
@@ -0,0 +1,760 @@
+id,name,sameAs,sameAsNote
+ocd-division/country:et,Ethiopia,,
+ocd-division/country:et/administration:aa,Addis Ababa,,
+ocd-division/country:et/administration:aa/ed:44075,44075,,
+ocd-division/country:et/administration:aa/ed:werda_1_and_9,Werda 1/9,ocd-division/country:et/administration:aa/ed:woreda_1_and_9,Possible spelling change across terms

```

Comment:
Are the alternative spellings necessary? We typically only create "sameAs" entries when renaming an OCD-ID – not when creating it.

Reply:
Got you, thank you for that, so something that may need more clarification would be how to capture
instances where a region has multiple versions of the name eg [ Mekelle / Mekele / Mekale ](https://en.wikipedia.org/wiki/Mekelle)

Reply:
I think in this project we just adopt one name. There are many projects, like GeoNames, which track alternative names, but that is not the goal of this project.

Reply:
Alright, this will be cleaned up and updated.